src/app/(proper_react)/(redesign)/(authenticated)/admin/dev/actions.tsx

```diff
@@ -93,13 +92,7 @@ export async function getAllProfileScans(onerepProfileId: number) {
 
 export async function triggerManualProfileScan(onerepProfileId: number) {
   const session = await getServerSession();
-  if (
-    !session?.user?.email ||
-    !isAdmin(session.user.email) ||
-    // only allow admins to trigger a scan for their own profile in production
-    (process.env.APP_ENV === "production" &&
-      session.user.subscriber?.onerep_profile_id !== onerepProfileId)
-  ) {
+  if (!session?.user?.email || !isAdmin(session.user.email)) {
```

Comment:
And maybe here a check on email address as well? Although I think this one maybe doesn't expose PII and can be helpful for customer support?

Reply:
I’ve added the additional check for now. See https://github.com/mozilla/blurts-server/pull/5850#discussion_r2068606190.